### PR TITLE
Add new param  for Yii::setAlias() enh #13957

### PR DESCRIPTION
**Pull request by unasoft**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #13957

### CODE REVIEW

File: framework/BaseYii.php [hunk 1]
```diff
@@ -201,6 +201,7 @@ public static function getRootAlias($alias)
      * alias translation by [[getAlias()]].
      * @param string $path the path corresponding to the alias. If this is null, the alias will
      * be removed. Trailing '/' and '\' characters will be trimmed. This can be
+     * @param boolean $overwrite the overwrite allow to overwrite the alliance if it is set, or skip if the alias is already set
```
Comment (Kolyunya) [inline framework/BaseYii.php]: > overwrite the alliance

`alias`?

Comment (ejsoft) [inline framework/BaseYii.php]: yeap, thanks

File: framework/BaseYii.php [hunk 2]
```diff
@@ -210,11 +211,14 @@ public static function getRootAlias($alias)
      * @throws InvalidParamException if $path is an invalid alias.
      * @see getAlias()
      */
-    public static function setAlias($alias, $path)
+    public static function setAlias($alias, $path, $overwrite = true)
```
Comment (samdark) [inline framework/BaseYii.php]: Public method signature change is a backwards compatibility break.

Comment (samdark) [inline framework/BaseYii.php]: I don't think we need it as option. It should be default behavior not to overwrite it.

**Review by samdark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**